src/ONYXKEYS.ts

```diff
@@ -291,6 +291,9 @@ const ONYXKEYS = {
     /** Indicates whether an forced upgrade is required */
     UPDATE_REQUIRED: 'updateRequired',
 
+    /** Indicates whether an forced reset is required a.k.a. clear Oynx data without signing the user out */
```

Comment:
Do you think it would be a good idea to include the intention for this feature in the comment? If it's meant to be debug-only, maybe we could highlight it

Reply:
Given the confusion experienced here I think it's a great idea to update this and explain the intention! 

It's a server side way to force clients to reload in case some backend code leaves the app in a broken state.

Reply:
Please let me know if the change looks good to you 🙇

Reply:
Looks much better! The comment doesn't mention that it's a server-to-client pipe, but maybe that's obvious.